DayNightQ10.Rmd

```diff
@@ -148,21 +171,26 @@ for(rhds in rh_dataset_names){
       depth <- -1
     }
     dsd %>% 
-      group_by(YEAR, WOY, DAY_NIGHT) %>% 
+      group_by(YEAR, WOY, DAY_NIGHT, Rs_Rh) %>% 
       summarize(DATASET = rhds, 
                 DEPTH = depth, 
-                Q10 = calc_q10(temp, CSR_FLUX_CO2), 
-                TEMP = mean(temp, na.rm = TRUE), 
+                Q10 = calc_q10(temp, CSR_FLUX_CO2)[1],
+                R_SQUARED = calc_q10(temp, CSR_FLUX_CO2)[2],
+                P_VALUE = calc_q10(temp, CSR_FLUX_CO2)[3],
```

Comment:
Is there a way to make it so calc_q10 doesn't have to be called multiple times?

Reply:
Oh yes, let's change that. One way, since you're already using `dplyr::summarise`, is described by the last post [here](https://stackoverflow.com/questions/22240816/dplyr-summarise-with-multiple-return-values-from-a-single-function).